osquery/core/watcher.cpp

```diff
@@ -402,6 +402,8 @@ void WatcherRunner::stopChild(const PlatformProcess& child) const {
 
 PerformanceChange getChange(const Row& r, PerformanceState& state) {
   PerformanceChange change;
+  change.footprint = 0;
+  change.parent = 0;
```

Comment:
Why not do all 4 members?

Reply:
`change.iv` and `change.sustained_latency` are initialized on line 409/447.

Reply:
I'd rather be consistent and initialize everything early, one way is to add a constructor to the structure or use the initializer-list form `{0}` for the member definitions. Or just set the other two members `= 0` here.